Missing semicolon on d2c_messaging_ut.cpp

There is a missing semicolon on line 241

https://github.com/Azure/iot-hub-device-update/blob/bf2df7d3b7325ca068f8685c5eaa875dc013bdac/src/utils/d2c_messaging/tests/d2c_messaging_ut.cpp#L241-L242

That causes the build to fail when trying to compile using catch2 v3.8.0.

```sh
-- Build files have been written to: /workspaces/iot-hub-device-update/out
[118/271] Building CXX object src/utils/d2c_messaging/tests/CMakeFiles/d2c_messaging_unit_test.dir/d2c_messaging_ut.cpp.o
FAILED: src/utils/d2c_messaging/tests/CMakeFiles/d2c_messaging_unit_test.dir/d2c_messaging_ut.cpp.o 
/usr/bin/g++ -DADUC_LOG_FOLDER=\"/var/log/adu\" -DADUC_USE_ZLOGGING=1 -DARCHITECTURE_x86_64=1 -I/usr/local/lib/cmake/inc -I/usr/local/lib/cmake/pal/linux -I/workspaces/iot-hub-device-update/src/utils/d2c_messaging/tests/inc -I/workspaces/iot-hub-device-update/src/inc -I/workspaces/iot-hub-device-update/src/adu_types/inc -I/workspaces/iot-hub-device-update/src/communication_abstraction/inc -I/workspaces/iot-hub-device-update/src/utils/c_utils/inc -I/workspaces/iot-hub-device-update/src/logging/inc -I/workspaces/iot-hub-device-update/src/logging/zlog/inc -I/workspaces/iot-hub-device-update/src/utils/d2c_messaging/./inc -I/workspaces/iot-hub-device-update/src/utils/retry_utils/./inc -I/workspaces/iot-hub-device-update/src/libaducpal/inc -Werror -D_FILE_OFFSET_BITS=64 -fPIE -D_FORTIFY_SOURCE=2 -fstack-protector-strong -Wformat -Werror=format-security -fno-rtti -g -std=gnu++17 -MD -MT src/utils/d2c_messaging/tests/CMakeFiles/d2c_messaging_unit_test.dir/d2c_messaging_ut.cpp.o -MF src/utils/d2c_messaging/tests/CMakeFiles/d2c_messaging_unit_test.dir/d2c_messaging_ut.cpp.o.d -o src/utils/d2c_messaging/tests/CMakeFiles/d2c_messaging_unit_test.dir/d2c_messaging_ut.cpp.o -c /workspaces/iot-hub-device-update/src/utils/d2c_messaging/tests/d2c_messaging_ut.cpp
/workspaces/iot-hub-device-update/src/utils/d2c_messaging/tests/d2c_messaging_ut.cpp: In function ‘int MockMessageTransportFunc(void*, void*, ADUC_C2D_RESPONSE_HANDLER_FUNCTION)’:
/workspaces/iot-hub-device-update/src/utils/d2c_messaging/tests/d2c_messaging_ut.cpp:242:5: error: expected ‘,’ or ‘;’ before ‘g_c2dResponseHandlerFunc’
  242 |     g_c2dResponseHandlerFunc = c2dResponseHandlerFunc;
      |     ^~~~~~~~~~~~~~~~~~~~~~~~
[131/271] Building CXX object src/utils/jws_utils/tests/CMakeFiles/jws_utils_unit_test.dir/jws_utils_ut.cpp.o
ninja: build stopped: subcommand failed.
```

--- a/src/utils/d2c_messaging/tests/d2c_messaging_ut.cpp
+++ b/src/utils/d2c_messaging/tests/d2c_messaging_ut.cpp
@@ -238,7 +238,7 @@ int MockMessageTransportFunc(
     void* cloudServiceHandle, void* context, ADUC_C2D_RESPONSE_HANDLER_FUNCTION c2dResponseHandlerFunc)
 {
     UNREFERENCED_PARAMETER(cloudServiceHandle);
-    INFO("Message transport function called.")
+    INFO("Message transport function called.");
     g_c2dResponseHandlerFunc = c2dResponseHandlerFunc;
     CAPTURE(g_c2dResponseHandlerFunc);
     auto message_processing_context = static_cast<ADUC_D2C_Message_Processing_Context*>(context);

